linux_os/guide/system/permissions/partitions/mount_option_nodev_nonroot_local_partitions/tests/correct.pass.sh

```diff
@@ -14,10 +14,10 @@ for partition in ${partitions[@]}; do
     mount -o remount "$partition"
 done
 
-PARTITION="/dev/new_partition1"; create_partition
+PARTITION="/new_partition1"; create_partition
```

Comment:
I suggest to open another PR for changes in this commit and keep this PR only for changes related to `mkdir -p`.